### PR TITLE
Update luma.gl to 8.4.5

### PR DESCRIPTION
**Pull request by zakjan**
See https://github.com/visgl/luma.gl/pull/1469

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "@deck.gl/core": "^8.2.0",
     "@deck.gl/layers": "^8.2.0",
     "@deck.gl/mesh-layers": "^8.2.0",
-    "@luma.gl/constants": "^8.2.0",
+    "@luma.gl/constants": "^8.4.5",
     "@luma.gl/core": "^8.2.0",
     "@luma.gl/engine": "^8.2.0",
```
Comment (kylebarron) [inline package.json:44]: Have you tested out this change? Usually if you have `luma.gl/core` 8.2 and `luma.gl/constants` 8.4, luma will raise an error when it detects multiple versions.

Comment (kylebarron) [reply package.json:44]: I don't remember exactly what is inherited from `BitmapLayer` but for https://github.com/kylebarron/deck.gl-raster/pull/61 do you need deck.gl to be `8.4+`?

Comment (zakjan) [reply package.json:44]: Hmm, not sure, let's update them all then? peerDependencies actually don't apply when a global build is used, so this bump is good only for reference.

Comment (kylebarron) [reply package.json]: I don't have a problem to updating to require 4.5+

Comment (zakjan) [reply package.json]: Ok, good

**Review by kylebarron — APPROVED** [review]

**Pull request merged**